docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md

```diff
@@ -112,6 +129,97 @@ To learn how to apply the changes, review [Apply Profile Updates to Clusters](#a
+  [Cluster Profile permissions](../../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile-admin)
+  reference for more information about roles and permissions.
+
+### Review Update Changes
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. From the left **Main Menu**, select **Profiles**.
+
+3. Select the profile you want to update to access the profile details page.
+
+4. Palette displays profile details and the profile stack. If there are pending updates, Palette displays a green
+   **Update** button in the top right-hand corner of the page. Click on the **Update** button to view the changes
+   summary modal.
+
+5. If the changes can be applied without any issues, then Palette will display the **Apply Changes** button.
+
+   ![A view of the cluster profile update widget displaying a new version of Calico is available.](/profiles_cluster-profiles_modify-cluster-profiles_new-version-notifcation.webp)
+
+   Otherwise, you will be presented with the **Review changes in Editor** button, which allows you to review the changes
+   before applying them.
+
+   :::tip
+
+   If a pack row has an information icon, hover over the icon to learn more about the changes.
+
+   :::
+
+   ![A view of the cluster profile update widget displaying a new packs versions but changes that require the user's input.](/profiles_cluster-profiles_modify-cluster-profiles_new-version-notifcation-changes-required.webp)
+
+6. Click on the **Apply Changes** button to apply the updates to the profile. If have changes that require your
+   attention then click on the **Review changes in Editor** button to start the review process.
+
+7. The differential editor will display the changes between the current YAML configuration and new incoming YAML
+   changes. The left side of the editor displays the current configuration. The right side displays the new pack
+   version's incoming changes. Review the changes and apply them as needed. Use the three buttons at the bottom to
+   navigate through the changes.
+
+   - **Prev**: Click to navigate to the previous change.
+   - **Next**: Click to navigate to the next change.
+   - **Keep**: Click to apply the current change.
+   - **Revert**: Click to revert the accepted change. This button will only appear after you have clicked the **Keep**
+     button.
+
+   The differential editor will display the changes between the current YAML configuration and new incoming YAML changes
+   by highlighting the differences. The color-coded highlights indicate the following:
+
+   - _Yellow highlight_ indicates text that is not present in the new configuration. These may be lines you have added
+     in the current configuration. These lines may also have been removed because they are no longer valid in the new
+     configuration. If you need them, use the **Keep** button to transfer the lines to the new pack version. Otherwise,
```

Comment:
```suggestion
   - _Yellow highlight_ indicates text that is not present in the new configuration. These may be lines you have added
     in the current configuration or lines that have been removed because they are no longer valid in the new
     configuration. If you need them, use the **Keep** button to transfer the lines to the new pack version. Otherwise,
```